spring-cloud-starter-dataflow-server-local/src/test/resources/org/springframework/cloud/dataflow/server/local/security/oauthConfig.yml

```diff
@@ -1,6 +1,3 @@
-management:
-  security:
-    enabled: true
```

Comment:
Interesting point I will add a test suite with that enabled to have that test-covered. I will add that once I brought the kids to school.